scripts/cbmc_json_parser.py

```diff
@@ -373,19 +374,27 @@ def has_check_failure(properties, message):
 def filter_reach_checks(properties):
     return filter_properties(properties, GlobalMessages.REACH_CHECK_DESC)
 
+def filter_sanity_checks(properties):
+    return filter_properties(properties, None, "sanity_check")
 
-def filter_properties(properties, message):
+def filter_properties(properties, message=None, property_class=None):
```

Comment:
Do we want a single `property_class` or a list?

Reply:
Maybe a wrapper function over this function which takes in a list of property_classes and messages? For this function itself, I think it's better to keep it simple, and call it over a list somewhere else.